Sharpmake.Generators/VisualStudio/Csproj.cs

```diff
@@ -1198,7 +1198,7 @@ List<string> skipFiles
             string projectConfigurationCondition = Template.Project.DefaultProjectConfigurationCondition;
             if (isNetCoreProjectSchema)
             {
-                netCoreEnableDefaultItems = "false";
+                netCoreEnableDefaultItems = project.EnableDefaultItems.ToString();
```

Comment:
iirc the boolean ToString() will return "True" or "False", so please call .ToLowerInvariant() on the resulting string so it's a valid xml value

Reply:
indeed, good catch.